Description: fix the manpage (hyphen as minus sign)
 (only a band-aid; real fix would mean rewriting this in mdoc)
Author: mirabilos <tg@debian.org>
Forwarded: https://github.com/AntiMicro/antimicro/pull/274

--- a/other/antimicro.1
+++ b/other/antimicro.1
@@ -25,7 +25,7 @@ output version information and exit
 \fB\-\-profile\fR \fI<location>\fR
 use specified profile as default for selected controllers. Defaults to all controllers.
 .TP
-\fB\-\-profile-controller\fR \fI<value>\fR
+\fB\-\-profile\-controller\fR \fI<value>\fR
 apply configuration file to a specific controller. <value> can be an controller index, name, or GUID.
 .TP
 \fB\-\-hidden\fR
